CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   BaseRequest and BaseResponse interface added which will be used inside recipe instead of previously used express.Request and express.Response
 -   `framework` config option. Default value is `express`.
 -   basic tests for all frameworks.
+-   Support for multiple access token signing keys
```

Comment:
Incorrect placement of this. Should be in 7.1.0. Likewise, change base branch to 7.1 for this PR